# 03.30

account 외부DB 죽이고 django-allauth로 구현했음.
내일은 contrib.auth랑 allauth 연결시켜야지
